### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'process.stderr.fd')

### ISSUE THREAD

**Issue by fernandolguevara** [oven-sh/bun#1518]
### What version of Bun is running?

0.2.2

### What platform is your computer?

macos m1 max arm64

### What steps can reproduce the bug?

```sh
mkdir buntest
cd buntest
echo -e "console.log(process.stderr.fd)" > index.ts
bun index.ts
```

### How often does it reproduce? Is there a required condition?

always

### What do you see instead?

1 | console.log(process.stderr.fd);
               ^
TypeError: undefined is not an object (evaluating 'process.stderr.fd')

**Comment by Electroid** [comment]
We're working on this right now! Tracking the issue in #894, and should be fixed when #1495 is merged.

**Comment by subhanahmed047** [comment]
Came across the same issue. Looking forward to the fix. 

```
bun index.ts 
153 |  */
154 | 
155 | function useColors() {
156 |   return 'colors' in exports.inspectOpts ?
157 |           Boolean(exports.inspectOpts.colors) :
158 |           tty.isatty(process.stderr.fd);
                ^
TypeError: undefined is not an object (evaluating 'process.stderr.fd')
```

For context, I get this when I try to require `sequalize` in an express app.
